src/utilcode/CMakeLists.txt

```diff
@@ -25,7 +24,6 @@ set(UTILCODE_SOURCES
   stgpool.cpp
   stgpooli.cpp
   stgpoolreadonly.cpp
-  lazycow.cpp
```

Comment:
This highlighted a mismatch between the old msbuild-based scripts and CMakefiles. This should be left in and FEATURE_LAZYCOW defined for x86 and arm instead (on Windows at least) - to match https://github.com/dotnet/coreclr/blob/master/clr.coreclr.props#L68.